{{ cookiecutter.repo_name }}/.pre-commit-config.yaml

```diff
@@ -2,13 +2,17 @@ default_language_version:
     python: python3
 repos:
   - repo: https://github.com/ambv/black
-    rev: stable
+    rev: 22.12.0
     hooks:
       - id: black
   - repo: https://github.com/pre-commit/pre-commit-hooks
     rev: v2.0.0
     hooks:
       - id: flake8
```

Comment:
With pre-commit-hooks v4+, this `flake8` option did not work for me. I think it should be updated. That's what I came up with: https://github.com/BNL-ATF/ophyd-basler/blob/main/.pre-commit-config.yaml. Could you please try that config?

Reply:
Good shout, better to be explicit and not introduce another dependency. Changed below.

Reply:
I also had a look over the other features in v4.4.0 and nothing else popped out as a "must" include.